src/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,103 @@
+import '../../index.css';
+
+import type { Metadata } from 'next';
+import type { ReactNode } from 'react';
+
+import Flyout from '@components/Flyout';
+import Header from '@components/Header/Header';
+import Providers from '@providers/Providers';
+import { hasLocale, NextIntlClientProvider } from 'next-intl';
+import { getTranslations, setRequestLocale } from 'next-intl/server';
+import { notFound } from 'next/navigation';
+
+import { routing } from '../../i18n/routing';
```

Comment:
```suggestion
import { routing } from '@i18n/routing';
```

To maintain a consistent and cleaner import style, it's a good practice to use alias imports for all local files where possible.